src/main/java/persistence/sql/ddl/EntityFields.java

```diff
@@ -0,0 +1,56 @@
+package persistence.sql.ddl;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+import persistence.sql.ddl.exception.IncorrectIdFieldException;
+import persistence.sql.ddl.exception.NotEntityException;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+
+public record EntityFields (String tableName, EntityIdField idField, List<EntityField> fields){
+    public static <T> EntityFields from(Class<T> clazz) {
+        if (!clazz.isAnnotationPresent(jakarta.persistence.Entity.class)) {
```

Comment:
```suggestion
        if (!clazz.isAnnotationPresent(Entity.class)) {
```
path 명시 하지 않고 import하여 사용해도 되지 않을까요?

Reply:
이부분 빼먹었군요; ㄷㄷ;;